datalad/support/annexrepo.py

```diff
@@ -2429,7 +2429,13 @@ def info(self, files, batch=False, fast=False):
         # and that they all have 'file' equal to the passed one
         out = {}
         for j, f in zip(json_objects, files):
-            assert(j.pop('file') == f)
+            # starting with version of annex around
+            # 8.20200330+git119-gcee6b344b
+            # annex started to normalize paths.
+            # https://github.com/datalad/datalad/issues/4431
+            # so we would use normpath around each side to assure it is the
+            # same file
+            assert(normpath(j.pop('file')) == normpath(f))
```

Comment:
While you're rewriting this line, would you mind dropping the odd parens (i.e. `assert(normpath...` -> `assert normpath...`)?